utils/PHPStan/CheckUseStatementsAfterLicenseRule.php

```diff
@@ -35,7 +35,7 @@ public function processNode(Node $node, Scope $scope): array
 	{
 		$comments = $node->getAttribute('comments');
 
-		if ($comments === [])
+		if ($comments === [] || ! is_array($comments))
```

Comment:
`$node->getAttribute('comments')` will always returns `array`, if PHPStan complain about it, we can add `@var` doc before assignment:

```php
                /** @var array $comments */
 		$comments = $node->getAttribute('comments');
```

Reply:
I didn't mean to commit that, but that's a local fix I had to make because the PHPStan pre-commit hook would fail with "Invalid use of foreach on non-array". I was planning to add it at some point so maybe now is the time. I'm not sure what is causing the issue higher but it is an actual type issue, not a static analysis issue.

Reply:
Ok, you can change above assignment to:

```php
$comments = $node->getComments();
```

Above will always returns array. Ref : 

https://github.com/nikic/PHP-Parser/blob/1d1bc8a36459d67acef508b952dad097960b110e/lib/PhpParser/NodeAbstract.php#L104-L106

that `getComments()` has empty `[]` fallback.

Reply:
Yep! That worked. Incoming.